tools/myst/host/exec.c

```diff
@@ -204,31 +185,24 @@ int exec_launch_enclave(
     return retval;
 }
 
-#define USAGE_EXEC_SGX \
-    "\
+#define USAGE_EXEC_SGX "\n\
 \n\
-Usage: %s exec-sgx <rootfs> [options] <application> <app_args...>\n\
+Usage: %s exec-linux [options] <rootfs> <application> <args...>\n\
 \n\
 Where:\n\
-    exec-sgx      -- execute the specified <application> from within the\n\
-                     <rootfs> with the <app_arguments> within an SGX enclave\n\
-    <rootfs>      -- This is the CPIO archive (created via mkcpio) of the\n\
-                     application directory\n\
-    <application> -- the application path from within <rootfs> to run within\n\
-                     the SGX enclave\n\
-    <app_args>    -- the application arguments to pass through to\n\
-                     <application>\n\
+    exec-linux    -- execute an application within <rootfs> in a trusted\n\
+                     SGX enclave\n\
+    <rootfs>      -- the root file system containing the application (CPIO or\n\
+                     EXT2)\n\
+    <application> -- the path of the executable program within <rootfs> that\n\
+                     will be executed\n\
+    <app_args>    -- arguments to passed through to the <application>\n\
 \n\
-and [options] are one or more of:\n\
-    --help                          -- this message\n\
-    --user-mem-size <size>          -- for running an unsigned binary this overrides the\n\
-                                       default user memory size for an application.\n\
-                                       The <size> format is a number that is in\n\
-                                       bytes (<size>), in kilobytes (<size>k), \n\
-                                       in megabytes (<size>m), or gigabytes (<size>g)\n\
-    --app-config-path <config.json> -- specifies the configuration json file for running an\n\
```

Comment:
Any reason we want to get rid of `--app-config-path`?

Reply:
Oh my! Thanks for finding this. It was a cut-n-paste error.